server/typings/tsd.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="node/node.d.ts" />
```

Comment:
Should this be versionned? Idem for `server/typings/node/node.d.ts` (which is relatively large).